cardano-cli/test/cardano-cli-golden/files/golden/help/allegra_query_kes-period-info.cli

```diff
@@ -24,6 +24,6 @@ Available options:
                            CARDANO_NODE_NETWORK_ID environment variable
   --testnet-magic NATURAL  Specify a testnet magic id. This overrides the
                            CARDANO_NODE_NETWORK_ID environment variable
+  --out-file FILEPATH      The output file.
   --op-cert-file FILEPATH  Filepath of the node's operational certificate.
-  --out-file FILEPATH      Optional output file. Default is to write to stdout.
```

Comment:
Why the change? It was more informative before.

Reply:
Reverted this to use the better message :+1:

Reply:
(the difference was between using `optional pOutputFile` and `pMaybeOutputFile`. The latter has the better message and I made sure to use it everywhere now)